### DISCUSSION TITLE
code is appended after the closing </html> tag.

### DISCUSSION THREAD

**Discussion by webbyben** [vercel/next.js#50480] [Show and tell] [answered]
When inspecting the page source of my website, I noticed that a significant amount of code is appended after the closing </html> tag. This additional code includes the entire content of the page, resulting in duplication.
`</body></html><script>(self.__next_f=self.__next_f||[]).push([0])</script><script>self.__next_f.push([1,"1:HL[\"/_next/static/media/2aaf0723e720e8b9-s.p.woff2\",{\"as\":\"font\",\"type\":\"font/woff2\"}]\n2:HL[\"/_next/static/css/app/layout.css?v=1685346700397\",{\"as\":\"style\"}]\n0:\"$L3\"\n"])</script><script>self.__next_f.push([1,"4:I{\"id\":\"(app-client)/./node_modules/next/dist/client/components/app-router.js\",\"chunks\":[\"webpack:static/chunks/webpack.js\"],\"name\":\"\",\"async\":false}\n6:I{\"id\":\"(app-client)/./node_modules/next/dist/client/components/error-boundary.js\",\"chunks\":[\"webpack:static/chunks/webpack.js\"],\"name\":\"\",\"async\":false}\n7:I{\"id\":\"(app-client)/./node_modules/next/dist/client/script.js\",\"chunks\":[\"app/layout:static/chunks/app/layout.js\"],\"name\":\"\",\"async\":false}\n8:I{\"id\":\"(app-client)/./src/components/Header.js\",\"chunks\":["])</script><script>self.__next_f.push([1,"\"app/layout:static/chunks/app/layout.js\"],\"name\":\"\",\"async\":false}\n9:I{\"id\":\"(app-client)/./node_modules/next/dist/client/image.js\",\"chunks\":[\"app/(site)/about/page:static/chunks/app/(site)/about/page.js\"],\"name\":\"\",\"async\":false}\na:I{\"id\":\"(app-client)/./node_modules/next/dist/client/link.js\",\"chunks\":[\"app/(site)/about/page:static/chunks/app/(site)/about/page.js\"],\"name\":\"\",\"async\":false}\nb:I{\"id\":\"(app-client)/./node_modules/next/dist/client/components/layout-router.js\",\"chunks\":[\"app-client-internals:st"])</script><script>self.__next_f.push([1,"atic/chunks/app-client-internals.js\"],\"name\":\"\",\"async\":false}\nc:I{\"id\":\"(app-client)/./node_modules/next/dist/client/components/render-from-template-context.js\",\"chunks\":[\"app-client-internals:static/chunks/app-client-internals.js\"],\"name\":\"\",\"async\":false}\n"])</script><script>self.__next_f.push([1,"3:[[[\"$\",\"link\",\"0\",`

Is it norma? If not, what is the fix?

**Comment by icyJoseph** [comment]
Original report: https://github.com/vercel/next.js/issues/48372
Fix: https://github.com/vercel/next.js/pull/50056

**Reply by webbyben** [reply]
@icyJoseph thanks. Please bear with me. Would you mind giving me some hints on what I should do to fix the issue?

**Reply by icyJoseph** [reply, marked as answer]
Wait until a new version comes out, or if you are really bothered, install next@canary, I think the patch is included there. Honestly I have not verified myself, other than through issue/PR tracking.

I had some thoughts on the root issue, but I'll reserve those.

**Reply by icyJoseph** [reply]
Verified in next@canary:

![Screenshot 2023-05-30 at 09 42 38](https://github.com/vercel/next.js/assets/21013447/1b205758-b6cd-4b1e-8749-2281ae710721)
